osscameroon/place
# Setup Scripts for docker #21

# Overview
This merge request introduces new scripts for easy environment setup for both Windows and Unix-like systems, along with a comprehensive update to the README file. These changes aim to streamline the initial setup process for the Place project, making it more accessible to contributors and users.

# Changes

## New Scripts Added

- `launch_project.ps1`: A PowerShell script for Windows users. It guides the user through setting up environment variables and updates the system's hosts file. The script requires administrative privileges to modify system settings.
- `launch_project.sh`: A Bash script for Unix-like systems. Similar to the PowerShell script, it assists in setting up environment variables and modifies the /etc/hosts file (sudo access required).

## README Update

The README has been thoroughly updated to include a new "Getting Started" section. This section provides step-by-step instructions on how to clone the repository, set up the environment, and launch the project using the new scripts.

Additional details have been added to guide the user in accessing the application once it's running and how to make contributions to the project.

## Testing

Both scripts have been tested in their respective environments (Windows 10 for PowerShell, Mac OS Ventura for Bash).

The steps in the README have been followed from start to finish to ensure they are clear and accurate.

## How to Test

1. Clone the feature branch.
2. Follow the "Getting Started" section in the README.
3. Verify that the environment setup completes successfully.
4. Ensure that the Place project runs as expected and is accessible via the provided URLs.

## Request for Review

I would appreciate a review of the changes, focusing on:

- The completeness and clarity of the README instructions.
- The functionality and user-friendliness of the setup scripts.

Feedback or suggestions for improvements are most welcome.
